themes/_core/js/checkout-customer-form.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+import $ from 'jquery';
+import prestashop from 'prestashop';
+
+export default function () {
+  const $body = $('body');
+  const {checkoutNewCustomerRelatedBlock} = prestashop.selectors.checkout;
+  const checkoutLoginAlertClass = 'js-checkout-login-alert';
```

Comment:
is there a reason for not putting this class in `themes/_core/js/selectors.js` and call it from there, as you did for `checkoutLoginAlertClass` ?

Reply:
(is it because of the dot ?)

Reply:
@matthieu-rolland yes, since it is not only selector but also class for alert block (alert is created dynamically via js). I could just place it inside `selectors.js` files and then remove dot but it might create problem in the future if someone will make nested selector for example.